webextensions/api/dom.json

```diff
@@ -0,0 +1,28 @@
+{
+  "webextensions": {
+    "api": {
+      "dom": {
+        "openOrClosedShadowRoot": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/dom/openOrClosedShadowRoot",
+            "support": {
+              "chrome": {
+                "version_added": false
```

Comment:
Supported since Chrome 88 - https://bugs.chromium.org/p/chromium/issues/detail?id=778816#c29

Which means that Opera 74 supports this too (https://en.wikipedia.org/wiki/History_of_the_Opera_web_browser#Opera_2021)

and Edge 88 supports this method too.